core-android/src/test/java/com/uber/sdk/android/core/RobolectricTestBase.java

```diff
@@ -30,7 +30,7 @@
 import org.robolectric.annotation.Config;
 
 @RunWith(RobolectricTestRunner.class)
-@Config(sdk = 21)
+@Config(sdk = 26)
```

Comment:
Might this also have unintended consequences in our CI?

Reply:
we don't consume this sdk internally. the min supported sdk version is 26 so should be fine.